active/tab/tabcontrol.md

```diff
@@ -0,0 +1,580 @@
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+TabView is a collection of tabs that each represents a new page or document in your app. Use TabView when your app has several pages of content and the user expects to be able to add, close, and rearrange the pages.
+
+**Is this the right control?**
+
+Use a TabView to help the user manage multiple app pages or documents within the same window. 
+
+Do not use a TabView to display a static set of tabs that the user cannot rearrange, open, or close. Use a Pivot or top NavigationView instead. 
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+## To replicate the behavior of Microsoft Edge
+
+``` xml
+<TabView TabViewItemWidthMode="Equal"
+            CanCloseTabs="True"
+            CloseButtonOverlay="OnHover"
+            CanDragTabs="True"
+            CanReorderTabs="True"
+            TabDraggedOutside="OpenTabInNewWindow"
+            AddTabButtonClick="NewTab_Click">
+    ...
+</TabView>
+```
+
+## Put TabView in the titlebar area
+
+![A sample app that shows how TabView can extend into the titlebar area](images/tab-extend-to-title.png)
+
+This sample demonstrates how to extend the TabView into the title bar area and also specify a portion of the UI as the draggable region, using [CoreApplicationViewTitleBar.ExtendViewIntoTitleBar](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationViewTitleBar.ExtendViewIntoTitleBar). Per the [titlebar customization guidelines](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar), we must set a [specific drag region](https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar#draggable-regions) (the space that a user can use to drag the window around the screen using mouse or touch). If we don't specify the drag region, the entire titlebar area remains draggable (meaning input won't be routed to the tabs, making the tabs unclickable).
+
+``` xml
+<Page>
+    <Grid>
+        <TabView>
+            <TabViewItem Icon="Home" Header="Home" IsCloseable="False" />
+            <TabViewItem Icon="Document" Header="Document 1" />
+            <TabViewItem Icon="Document" Header="Document 2" />
+            <TabViewItem Icon="Document" Header="Document 3" />
+        </TabView>
+
+        <Grid x:Name="CustomDragRegion" Width="200" Height="40" HorizontalAlignment="Right" VerticalAlignment="Top" />
```

Comment:
Presumably this drag region should be the `TabView.Footer`, so that tabs won't go into the drag region. Ideally, should use system metrics to obtain size of drag region, so we don't overlap system buttons.

How do we make things so that all the unused space in the TabView becomes drag region?

Reply:
You're correct, this is a sample bug. The custom drag region should be contained within TabView.TabStripFooter with alignment = stretch to fill the whole region. 

> Ideally, should use system metrics to obtain size of drag region, so we don't overlap system buttons.

I haven't found a good way to do this beyond setting a hardcoded min-size to the custom drag region. Is there a better way?

Reply:
You can find an example in the [TitleBar sample](https://github.com/microsoft/Windows-universal-samples/tree/master/Samples/TitleBar).

* [XAML](https://github.com/microsoft/Windows-universal-samples/blob/master/Samples/TitleBar/shared/CustomTitleBar.xaml)
* [C# code-behind](https://github.com/microsoft/Windows-universal-samples/blob/master/Samples/TitleBar/cs/CustomTitleBar.xaml.cs)

Reply:
Do we have guidelines the minimum size of the drag region?

Reply:
For some reason, the overlay inset is always returning 0 🤔  Following up offline.